readme.md

```diff
@@ -18,7 +18,7 @@ Plash enables you to have a highly dynamic desktop wallpaper. You could display
 - [**Random Unsplash image**](https://source.unsplash.com)\
 	Example: https://source.unsplash.com/random/2880x1756?puppy \
 		This returns a new random puppy image each time.
-		Replace the numbers with your screen resolution. Subtract 44 (or 22 for non-retina) on the height to account for the menu bar.
+		Use `[[screenWidth]]` and `[[screenHeight]]` in any URL and Plash will substitute the right values for you, e.g. `https://source.unsplash.com/random/[[screenWidth]]x[[screenHeight]]?puppy`.
```

Comment:
I think this should be documented in a separate section and we can just refer to it here. It's easy to miss when it's just here.

Reply:
Can I leave that up to you? I'm not sure where you'd want that information to live.

Reply:
Just add it to the Tip section for now: https://github.com/sindresorhus/Plash#tip

Reply:
And to https://github.com/sindresorhus/Plash/blob/master/app-store-description.txt

Reply:
Done.